lib/rubocop/cop/util.rb

```diff
@@ -193,6 +193,14 @@ def to_supported_styles(enforced_style)
           enforced_style.sub(/^Enforced/, 'Supported').sub('Style', 'Styles')
       end
 
+      def safe_parse_regexp(text)
```

Comment:
`safe_` might be redundant. `parse_regexp` is sufficient to convey the intent. It shouldn't be provided as there are no use cases, but if it were to raise exceptions, it could be contrasted with a `parse_regexp!` method.

```suggestion
      def parse_regexp(text)
```

Reply:
Yes, I agree with that. Renamed to `parse_regexp`